code-challenges.test.js

```diff
@@ -15,42 +15,132 @@
+      expect(singleLetter(arrayOfWords2, letterE)).toEqual(["Cherry", "Blueberry", "Peach"])
+    })
+  })
+
+// RED Fail:   ● singleLetter › returns a coded message. The coded message converts 'a' to 4, 'e' to 3, 'i' to 1, and 'o' to 0.
+
+    // ReferenceError: singleLetter is not defined
+
+
+
 // Expected output: ["Apple", "Banana", "Orange"]
-const arrayOfWords2 = ["Mango", "Cherry", "Apricot", "Blueberry", "Peach"]
-const letterE = "e"
+
 // Expected output: ["Cherry", "Blueberry", "Peach"]
 
 
 // b) Create the function that makes the test pass.
+// Make a function called singleLetter with array and string as param
+// Use the .filter method to filter the array
+// Make an empty array to hold the new value (Didn't work)
+// Make a conditional statement to see if the string equals to the filtered array (Didn't work)
+// Retun the array.filter to with a value that lower cases the words, and includes the stringed letter
 
+const singleLetter = (array, string) =>{
+    return array.filter(value => value.toLowerCase().includes(string))
+}
+
+//   singleLetter
+// ✓ takes in an array of words and a single letter and returns all the words that contain that particular letter.
 
 // --------------------3) Create a function that takes in an array of 5 numbers and determines whether or not the array is a “full house”. A full house is exactly one pair and one three of a kind.
 
 // a) Create a test with expect statements using the variable provided.
 
-const hand1 = [5, 5, 5, 3, 3]
+describe("pokerFace", () => {
+    const hand1 = [5, 5, 5, 3, 3]
+    const hand2 = [5, 5, 3, 3, 4]
+    const hand3 = [5, 5, 5, 5, 4]
+
+    it("Takes in an array of 5 numbers and determines whether or not the array is a “full house”. A full house is exactly one pair and one three of a kind.", () => {
+      expect(pokerFace(hand1)).toEqual(true)
+      expect(pokerFace(hand2)).toEqual(false)
+      expect(pokerFace(hand3)).toEqual(false)
+
+    })
+  })
+
 // Expected output: true
-const hand2 = [5, 5, 3, 3, 4]
 // Expected output: false
-const hand3 = [5, 5, 5, 5, 4]
 // Expected output: false
 
+// RED Fail: ● pokerFace › Takes in an array of 5 numbers and determines whether or not the array is a “full house”. A full house is exactly one pair and one three of a kind.
+
+    // ReferenceError: pokerFace is not defined
+
 
 // b) Create the function that makes the test pass.
+// Make a function called pokerFace with array as the param
+// Make a conditional statement to check if the arrays are true or false
+// In the conditional I'll have the array equal to an index of a number. If there's five numbers then I have atleast five solutions
+// The if statement is just going to contain all the possiblities to keep it simple and it'll return true
+// The else will return false
+
+const pokerFace = (array) => {
+    if  (array[0] === array[1,2] && array[3] === array[4] ||
+         array[1] === array[2,3] && array[4] === array[1] ||
```

Comment:
Your tests are definitely passing, but this may not work for other variations of the test variables.  You are checking for array[1] on both the set of 3 and 2 on this line.